lib/rules/block-no-empty/README.md

```diff
@@ -9,6 +9,8 @@ Disallow empty blocks.
  * Blocks like this */
 ```
 
+The [`message` secondary option](../../../docs/user-guide/configure.md#message) can accept the arguments of this rule.
```

Comment:
[ask] I wrote as below on <https://github.com/stylelint/stylelint/issues/6966#issue-1772346486>,

> Remove the sentence from all the rule README:

so this added line is unneeded, right?

Reply:
Ah yes, this was to make the tests pass (for now). My hope is to have this entire PR resolve #6966, in which I'll then remove the test + this line in all of the rule docs in one commit. But if not for whatever reason, then I'll keep this in for now.